test/web-platform-tests/to-upstream.js

```diff
@@ -1,8 +1,37 @@
 "use strict";
 const path = require("path");
-const readdirRecursive = require("fs-readdir-recursive");
```

Comment:
Can we remove this devDependency from package.json now?

Kind of a shame to lose the ease of use here, but maybe in the future I'll move toward pretest manifest generation.